bodhi/tests/server/views/test_generic.py

```diff
@@ -414,26 +414,6 @@ def test_version(self):
         res = self.app.get('/api_version')
         self.assertIn('version', res.json_body)
 
-    def test_new_override_form(self):
-        """Test the New Override form page"""
```

Comment:
Is this being tested somewhere else?

Reply:
yeah, the same tests were being done in test_overrides.py (where all the other tests were being done for the overrides pages too)

This test i removed was doing the same checks as these two:
https://github.com/fedora-infra/bodhi/blob/a4c4d756363694b07b89fd9e4ae6db09775b506b/bodhi/tests/server/services/test_overrides.py#L731

https://github.com/fedora-infra/bodhi/blob/a4c4d756363694b07b89fd9e4ae6db09775b506b/bodhi/tests/server/services/test_overrides.py#L746

Reply:
Cool, thanks